api/rest/restapi.go

```diff
@@ -263,6 +263,10 @@ func (api *API) addRoutes(router *mux.Router) {
 				),
 			)
 	}
+	router.NotFoundHandler = ochttp.WithRouteTag(
+		http.HandlerFunc(api.notFoundHandler),
+		"/"+"notfound",
```

Comment:
you must mean `"/notfound"` or `"/404"` (better), but not `"/"+"notfound"`

Reply:
```suggestion
		"/notfound",
```